lib/Runtime/Language/InterpreterLoop.inl

```diff
@@ -157,14 +157,25 @@ SWAP_BP_FOR_OPCODE:
                 return nullptr;
 
             case INTERPRETER_OPCODE::ExtendedOpcodePrefix:
+            case INTERPRETER_OPCODE::DblExtendedOpcodePrefix:
```

Comment:
@commonlisp , can you please take care of this along with having a separate case arm for DblExtendedOpcodePrefix ?